src/renderer/viz/parser.js

```diff
@@ -115,7 +119,7 @@ function parseFunctionCall (node) {
 function parseBinaryOperation (node) {
     const left = parseNode(node.left);
     const right = parseNode(node.right);
-    switch (node.operator) {
+    switch (node.operator.toLowerCase()) {
```

Comment:
So now we accept `and` and `AND` but also `And`, `aNd`, `anD`, `ANd`, `AnD` and `aND`. Oh well, I guess that's case-insensitivity is all about...

Reply:
hahaha yes 🤷‍♂️

Reply:
I thought we were going just after 'and' & 'AND' variations, so maybe we could add this on the case side (and remove `toLowerCase`):

```
    case 'and':
    case 'AND':
        return expressions.and(left, right);
    case 'or':
    case 'OR':
        return expressions.or(left, right);
    case 'in':
    case 'IN':
        return expressions.in(left, right);
    case 'nin':
    case 'NIN':
        return expressions.nin(left, right);
```